experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -14,26 +14,18 @@ class NimbusFmlLoader:
     BASE_PATH = Path(BASE_DIR, "features", "manifests", "apps.yaml")
     MANIFEST_PATH = Path(BASE_DIR, "features", "manifests")
```

Comment:
These you'll want to move into `settings.py`

Reply:
`BASE_PATH` is removed here entirely since it's no longer needed now that @brennie has landed the local manifests.

`MANIFEST_PATH` is staying here for now in this PR, and the work to override the settings that we discussed will be in #9768